apps/spruce/src/pages/waterfall/caching/index.ts

```diff
@@ -107,6 +110,7 @@ export const readVersions = ((existing, { args, readField }) => {
   return {
     flattenedVersions,
     pagination: {
+      activeVersionIds: activeVersionIds.sort(),
```

Comment:
is the sort just for guaranteeing consistency in testing? Could we add a comment if this is the case that the sort isn't inherently meaningful :0